modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4549,7 +4548,7 @@ input StepConfigInput {
   """
   Science step creation option
   """
-  science: StepConfigScienceInput
+  science: Boolean
```

Comment:
Why boolean?

Reply:
It marks that you want to create a science step.  Since there is no other state specific to a science step, there's nothing to configure but somehow you have to specify that you want a science step.